### DISCUSSION TITLE
Microwin Documentation and Debloat Details?

### DISCUSSION THREAD

**Discussion by uses0ap** [ChrisTitusTech/winutil#2718] [Q&A]
Pardon my ignorance, but I can’t seem to find any detailed documentation for Microwin. 
Is there any available that provides more specifics on what Microwin removes or changes? 

I’m particularly interested in details about:

- Hardware requirement removal
- Secure Boot removal
- TPM requirements removal
- BitLocker disabled

Additionally, it would be helpful to have a list of apps that are removed as part of the debloat process.

**Comment by beng-tang** [comment]
For microwin documentation, see: https://winutil.christitus.com/userguide/microwin/

**Reply by ghostinushanka** [reply]
Documentation does not answer the question.
I came here with the same after looking at the docs.

"remove telemetry & tracking" and "app debloat" is way too abstract. What if there is an app removed I am interested in having? This way I (as a user) cannot make an informed decision. And yes, project is opensource, but the whole point of making a tool for not-so tech savvy users is to provide a working solution to someone who probably cannot read the source to figure out stuff, right?

**Reply by Oldgamesenjoyer** [reply]
I agree with ghostinushanka, there should be an easy to read list of features removed
